src/MessageBox.Avalonia/BaseWindows/Base/IMsBoxWindow.cs

```diff
@@ -1,7 +1,7 @@
-using System.Threading.Tasks;
 using Avalonia.Controls;
+using System.Threading.Tasks;
 
-namespace MessageBox.Avalonia.BaseWindows
+namespace MessageBox.Avalonia.BaseWindows.Base
```

Comment:
This is breaking changes

Reply:
Here I think we need to converse with repository owner. If our point to provide API with a minimum count of breaking changes we can roll this back

Reply:
Yes, that's right. If this happens, then it needs to be mentioned somewhere to facilitate the transition for library users